docs/fieldnorm/control_of_documents.md

```diff
@@ -1,25 +1,91 @@
-5. changes and the current revision status of documents are identified;
-6. documents are protected from unauthorized changes and any deletion or removal;
-7. documents are protected from unauthorized access;
-8. the unintended use of obsolete documents is prevented, and suitable identification is applied to them if they are retained for any purpose;
-9. at least one paper or electronic copy of each obsolete controlled document is retained for a specified time period or in accordance with applicable specified requirements.
+As specified in the ISO-15189, "Software procedures and related documentation needs to be controlled". This can be in any form or type of medium, as long as a certain set of requirements is achieved. Software are developped and distributed using a version control system, such as git or gitlab. Those often offer extra functionnalities related to code (e.g. : linking a new release with a set of solved issues), which make them good candidate to be a _Bioinformatician_ QMS system. In this document, we offer guidelines on how to comply with the norms while implementing a _bioinformatician_ friendly QMS. 
 
-Examples are:
+## ISO norm requirements
+
+To allow for a "_Bioinformatician_ friendly" QMS, the following aspects need to be considered and documented :
+
+1. Documents are uniquely identified
+2. Documents are approved for adequacy before issue by authorized personnel who have the expertise and competence to determine adequacy
+3. Documents are periodically reviewed and updated as necessary
+4. Relevant versions of applicable documents are available at points of use and, where necessary, their distribution is controlled
+5. Changes and the current revision status of documents are identified
+6. Documents are protected from unauthorized changes and any deletion or removal
+7. Documents are protected from unauthorized access
+8. The unintended use of obsolete documents is prevented, and suitable identification is applied to them if they are retained for any purpose
+9. One paper or electronic copy of each obsolete controlled document is retained for a specified time period or in accordance with applicable specified requirements
+
+## Candidates for a Bioinformatician friendly QMS
+
+_Bioinformaticians_ create workflows collaboratively on a computer within a git environment. Git is a distributed version control system that tracks changes in any set of computer files. This can be used to track the changes of the code itself, but also in its documentation, which can describe the Standard Operating Procedures (SOP) to use it. Git comes in with a set of features allowing to track issues and their resolution, new release of a workflow and its newest features. As the workflow often get upgraded and updated, their documentation, as well, is susceptible to change. Which is why many chose _Bioinformaticians_ their own form of QMS. Examples of those documentations systems are:
```

Comment:
which can describe the Standard Operating Procedures (SOP) to use it. --> not a very clear sentence. Do you want to say that we need a procedure for using git, or that you can use git as a QMS for storing all procedures and hence SOPs